docs/en/relying-party-solution.rst

```diff
@@ -0,0 +1,308 @@
+    "kid": "e0bbf2f1-8c3a-4eab-a8ac-2e8f34db8a47",
+    "trust_chain":[
+      "MIICajCCAdOgAwIBAgIC...awz",
+      "MIICajCCAdOgAwIBAgIC...2w3",
+      "MIICajCCAdOgAwIBAgIC...sf2"
+    ]
+  }
+  .
+  { 
+    "transactionId": "e9673f90-a652-4c47-8039-f60096cabafb",
+    "state": "a1252f90-a652-4c47-8039-f60096cavzqw",
+    "nonce": "o1923f90-a652-4c47-8039-f60096cabcza",
+    "requestUri": "oid4vp-wallet://authorize?client_id=https://<VERIFIER_RELYING_PARTY_HOST>&request_uri=https://<VERIFIER_RELYING_PARTY_HOST>/oid4vp/a1252f90-a652-4c47-8039-f60096cavzqw",
+    "iss": "https://<VERIFIER_RELYING_PARTY_HOST>",
+    "aud": "https://<VERIFIER_RELYING_PARTY_HOST>",
+    "iat": "1686823112",
+    "exp": "1686823812"
+  }
+  .
+  SIGNATURE
+
+
+Header
+^^^^^^
+
+.. list-table:: 
+  :widths: 25 50
+  :header-rows: 1
+
+  * - **Param**
+    - **Description**
+  * - **alg**
+    - Algorithm used to sign the JWT
+  * - **typ**
+    - Media Type of the JWT
+  * - **x5c**
+    - X.509 Certitifcate Chain containing the Key used to sign the JWT
+  * - **kid**
+    - Key ID used identifying the Key used to sign the JWT. Required if trust_chain is used
+  * - **trust_chain**
+    - Sequence of verified Entity Statements that validates a participant's compliance with the Federation
+
+
+Payload
+^^^^^^^
+.. list-table:: 
+  :widths: 25 50
+  :header-rows: 1
+
+  * - **Param**
+    - **Description**
+  * - **transactionId**
+    - Unique identifier of the authorization transaction
+  * - **state**
+    - Unique identifier of the AuthorizationRequest
+  * - **nonce**
+    - Fresh cryptographically random number with sufficient entropy used for Security Reason
+  * - **requestUri**
+    - The payload of the QR Code from where the Wallet can obtain the Request Object containing Authorization Request parameters
+  * - **iat**
+    - The timestamp when the JWT was issued
+  * - **exp**
+    - The timestamp when the JWT is set to expire
+  * - **iss**
+    - The entity that issued the JWT. It will be populated with the Verifier URI
+  * - **aud**
+    - The audience of the JWT. It will be equal to iss value
+
+
+RequestObject Details
+---------------------
+The "requestUri" claim provided in AuthorizationRequest response object will be the payload of QR Code.
+The Wallet will invoke the Relying Party using the "request_uri" param found in QR Code in order to obtain the entire Authorization Request Object.
+
+.. code-block:: javascript
+  
+  GET /oid4vp/{id} HTTP/1.1
+  HOST: <VERIFIER_RELYING_PARTY_HOST>
+
+  HTTP/1.1 200 OK
+
+
+Below a non-normative response example
+
+.. code-block:: javascript
+
+  {
+    "alg": "ES256",
+    "typ": "JWT",
+    "x5c": [ "MIICajCCAdOgAwIBAgIC...20a" ],
+    "kid": "e0bbf2f1-8c3a-4eab-a8ac-2e8f34db8a47",
+    "trust_chain": [
+      "MIICajCCAdOgAwIBAgIC...awz",
+      "MIICajCCAdOgAwIBAgIC...2w3",
+      "MIICajCCAdOgAwIBAgIC...sf2"
+    ]
+  }
+  .
+  { 
+    "presentation_definition": null,
```

Comment:
following today's SIOP call 

there are some bad feelings about the misuse of the scope parameter

I'll continue working on that with the help of the authors and in the same time we need for the national solution a PoC with a `presentation_definition` for interop

presentation_definition could be aliased and embedded in separate scopes, it's challenging and interesting but still not ready

while we'll implement `presentation_definition` we'll continue working on a proposal

@TeamSog @grausof ^

Reply:
An example of presentation_definition is added in addition to scope, can you pls check it? @peppelinux